ui/pages/confirm-approve/confirm-approve-content/index.scss

```diff
@@ -9,10 +9,14 @@
     padding: 0 24px 16px 24px;
   }
 
-  &__unknown-asset {
+  &__approval-asset-link {
     color: var(--color-primary-default);
   }
 
+  &__approval-asset-title {
+    cursor: pointer;
```

Comment:
for case # 2, when the title does not have a link, can we disable the pointer?

Reply:
My thinking hear was to make the fact that you can click to copy the address (when there isn't a link) somewhat apparent. Probably could/should do the on-hover dropdown as well that Barbara highlights [here](https://github.com/MetaMask/metamask-extension/pull/15406#issuecomment-1201750793)

Reply:
Implementing the drop down is going to be a real pain... as noted in the comment linked above we are avoiding putting to much into this since we are going to implement a whole new design very soon?

Reply:
oh okay. I was confused by the various cases listed above. Sounds good to me to leave the pointer in and revisit this with the new redesign 👍